gradio/blocks.py

```diff
@@ -925,6 +925,7 @@ def __init__(
         js: str | None = None,
         head: str | None = None,
         fill_height: bool = False,
+        fill_width: bool = False,
```

Comment:
Why is this part of this PR? I thought this was already done in a separate PR?

Reply:
https://huggingface.slack.com/archives/C02QNREU24S/p1721340933971549